crimsobot/cogs/image.py

```diff
@@ -18,6 +20,27 @@ class Image(commands.Cog):
     def __init__(self, bot: CrimsoBOT):
         self.bot = bot
 
+    async def get_image_and_embed(self, ctx: commands.Context, image: Optional[str], effect: str,
```

Comment:
All of the image commands which rely on `utils/image/process_image()` route thru here to keep everything DRY. Filenames are made unique with a datetime string as per a user request.